econet_hvac_odu.yaml

```diff
@@ -252,6 +252,22 @@ sensor:
     request_mod: none
     accuracy_decimals: 0
     internal: true
+  - platform: econet
+    name: "ODU Size (Tons)"
+    id: odu_size
+    sensor_datapoint: ODU_SIZE
+    request_once: true
+    request_mod: 7
+    icon: "mdi:information-box"
+    entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Type"
+    id: odu_type
+    sensor_datapoint: ODU_TYPE
+    request_once: true
+    request_mod: 7
+    icon: "mdi:information-box"
+    entity_category: "diagnostic"
```

Comment:
What does this return? If it's 0 or 1 we should give it a more meaningful name and change it to be a `binary_sensor`. If it returns some string then it should be a `text_sensor`.

Reply:
> What does this return? If it's 0 or 1 we should give it a more meaningful name and change it to be a `binary_sensor`. If it returns some string then it should be a `text_sensor`.

I actually am not sure, as I don't have another ODU to gather data from.  Getting someone who has a non-communicating unit to report this datapoint, and someone with an RP20 [the full variable speed] unit would help solve this.

Reply:
If these are integer/ENUM values please set `accuracy_decimals: 0`

Reply:
There are 2.5 and 3.5 Ton sizes available.